### ISSUE TITLE
Accessibility?

### ISSUE THREAD

**Issue by fryford** [ONSvisual/subnational-indicators#14]
Some points to consider:

- Check that the tables read out OK.
- That we notify the user when an area is changed
- Maybe hide the graphic from view? Or replace with a hidden table for the area selected?
- Check contrast

**Comment by fryford** [comment]
Rob to build the table out in html

**Comment by rcatlord** [comment]
There are a bunch of errors e.g. lack of page regions and poor contrast judging by the [WAVE accessibility evaluation](https://wave.webaim.org/report#/https://onsvisual.github.io/subnational-indicators/app/index.html).

**Comment by henryjameslau** [comment]
Multiselect sorted by https://github.com/ONSvisual/subnational-indicators/pull/27

**Comment by henryjameslau** [comment]
![image](https://user-images.githubusercontent.com/2945099/143609498-840ebb99-cd31-46d5-bd4a-fd8619bc765f.png)
Ready for your second pair of eyes @giacomobg
